Policies/github-corporate-terms-of-service.md

```diff
@@ -25,23 +25,29 @@ Thank you for choosing GitHub for your company's business needs. Please read our
 | [O. Disclaimer of Warranties](#o-disclaimer-of-warranties) | We provide our service as is, and we make no promises or guarantees about this service. **Please read this section carefully; you should understand what to expect.** |
 | [P. Limitation of Liability](#p-limitation-of-liability) | Situations where either of us will be liable to the other for damages or losses arising from your use or inability to use the service or otherwise arising under this agreement are limited. **Please read this section carefully; it limits our obligations to you.** |
 | [Q. Release and Indemnification](#q-release-and-indemnification) | You are fully responsible for your and your Users' use of the service. |
-| [R. Changes to these Terms of Service](#r-changes-to-these-terms) | We may modify this agreement, but we will give you 30 days' notice of changes that affect your rights. |
-| [S. Miscellaneous](#s-miscellaneous) | Please see this section for legal details including our choice of law. |
+| [R. Confidentiality](#r-confidentiality) | We each have some responsibility to protect each other’s Confidential Information |
+| [S. Professional Services](#s-professional-services) | You may purchase Professional Services in connection with your use of the Website or Service. . |
```

Comment:
Typo: extra space and period not needed.

Reply:
```suggestion
| [S. Professional Services](#s-professional-services) | You may purchase Professional Services in connection with your use of the Website or Service. |
```

Thanks @meentastic!